recipe/meta.yaml

```diff
@@ -20,7 +20,7 @@ requirements:
     - pip
   run:
     - python >=3.6
-    - azure-core <2.0.0,>=1.15.0
+    - azure-core <2.0.0,>=1.23.1
     - cryptography >=2.1.4
     - msrest >=0.6.21
```

Comment:
https://github.com/Azure/azure-sdk-for-python/blame/azure-storage-blob_12.12.0/sdk/storage/azure-storage-blob/setup.py#L78-L82